### PR TITLE
Fix/learning progress

### PR DESCRIPTION
**Pull request by chfsx**
Provides a new Permission `read_learning_progress` to separate this from the permission `write`. Users with `write` still can see the LP as well.

### CODE REVIEW

File: classes/class.ilObjInteractiveVideoGUI.php [hunk 1]
```diff
@@ -2392,7 +2409,6 @@ public function postVideoStartedPerAjax()
 	{
 		global $ilUser;
 		$this->object->saveVideoStarted($this->obj_id, $ilUser->getId());
-		$this->object->trackProgress();
```
Comment (chfsx) [inline classes/class.ilObjInteractiveVideoGUI.php]: I has to remove this line to have a correct LP behaviour.

**Comment by gvollbach** [comment]
@chfsx Thanks for the work you invested here!

File: classes/class.ilObjInteractiveVideoGUI.php (hunk 1 above)
Comment (mjansenDatabay) [reply classes/class.ilObjInteractiveVideoGUI.php]: @chfsx What was the issue with this call?

P.S.: Thanks a lot for this PR!

Comment (chfsx) [reply classes/class.ilObjInteractiveVideoGUI.php]: LP did not work correct with this line, I wasn't able to complete an IV-Item

**Pull request merged**